src/domains/shared-app/api/status-page-service/v1/_base_policy.xml

```diff
@@ -29,11 +29,17 @@
                       return (string)services[context.Variables["product"]];
                   }" />
           <choose>
-            <when condition="@(context.Variables["baseUrl"] != null)">
-            <set-backend-service base-url="@{
-                          var services = JObject.Parse((string)context.Variables["services"]);
-                          return "https://" + (string)services[context.Variables["product"]];
-            }" />
+            <when condition="@(context.Variables["baseUrl"] != null && ((string)context.Variables["product"]).EndsWith("-fe"))">
+              <set-backend-service base-url="@{
+                                var services = JObject.Parse((string)context.Variables["services"]);
+                                return "https://" + (string)services[context.Variables["product"]] + "?";
```

Comment:
if th purpose is to add `?` or not., why not use `if` in to `{ ...}` code ?

Reply:
The '?' character is needed because seems that this policy was made in order to replace only the base URL. 
This character force to pass the section after the base path as a query parameter (that will be ignored during the redirect). I don't like it either but at the moment seems a way to not refactor the whole policy. Feel free to suggest a better way to do this!